src/main/java/vi/viredis/client/ViRedis.java

```diff
@@ -17,7 +17,12 @@
     private RedisServerWriter writer;
     private RedisServerReader reader;
 
+    private String host;
```

Comment:
Remove these two variable host and port from here. These host and port values are required just for creating a new connection in pool !!. This info is already available in RedisPool class. Whenever invalidation of old connection and creation of new connection is being done, just use  host and port info from RedisPool class.

Reply:
done!